arch/arm/core/aarch32/irq_manage.c

```diff
@@ -36,6 +36,90 @@ extern void z_arm_reserved(void);
 #define REG_FROM_IRQ(irq) (irq / NUM_IRQS_PER_REG)
 #define BIT_FROM_IRQ(irq) (irq % NUM_IRQS_PER_REG)
 
+#ifdef CONFIG_ZERO_LATENCY_IRQS_ARMV6_M
+/* initialise zero latency irq variables*/
+uint32_t zli_shadow_reg;
```

Comment:
these 3 variables can be marked static

Reply:
Thank you for the review. I will apply the suggested changes.